tests/test_git.sh

```diff
@@ -0,0 +1,59 @@
+
+# Error on unset variables
+set -u
+
+if [ -n "${ZSH_VERSION-}" ]; then
+  SHUNIT_PARENT="$0"
+  setopt shwordsplit
+fi
+
+. ../liquidprompt --no-activate
+
+function test_git {
+
+    LP_ENABLE_GIT=1
+    LP_ENABLE_FOSSIL=0
+    LP_ENABLE_SVN=0
+    LP_ENABLE_BZR=0
+    LP_ENABLE_VCS_ROOT=0
+    LP_ENABLE_VCS_REMOTE=1
+
+    PS1=""
+    lp_activate
+
+    wd="${SHUNIT_TMPDIR}/test_remote/"
+    mkdir -p "$wd"
+    cd "$wd"
+
+    _lp_are_vcs_enabled
+    assertTrue "VCS are enabled here." "$?"
+
+    git init
+    # We need a commit to have a branch that can have a remote.
+    touch test
+    git add test
+    git commit -m "test"
```

Comment:
This line is causing many problems. In CI, the `git` install is without config, so a commit throws an error because no author is set. Locally, I have commit signing enabled, so the tests halt to ask me for my key password.

Try this:
```suggestion
    git commit -m "test" --author="A U Thor <author@example.com>" --no-verify --no-gpg-sign
```